### ISSUE TITLE
Could gitlab support http?

### ISSUE THREAD

**Issue by imsobear** [paulirish/git-open#61]
RT

**Comment by ffes** [comment]
There is already an open pull request for this: #60

**Comment by noyobo** [comment]
@paulirish Please merge #60 asap

**Comment by MartinDelille** [comment]
@noyobo You can use this patch plugin in the meantime.

**Comment by paulirish** [comment]
I've just published 1.3.0 to npm which has a fix for this.


see https://github.com/paulirish/git-open#gitlab-support

```sh
# use --global to set across all repos, instead of just the local one
git config [--global] gitopen.gitlab.protocol http
```
